TFproviderTest.yml

```diff
@@ -171,29 +171,30 @@ pipelines:
             - cat GNUmakefile | sed -e "s/darwin_amd64/linux_amd64/g" > GNUmakefile.tmp
```

Comment:
If we add

```
TARGE_ARCH?=darwin_amd64
```
to `GNUmakefile`

And replace all instances of `darwin_amd64` with `${TARGE_ARCH}`

Then we won't need to do these sed and file gymnastic.

Just `make install -e TARGET_ARCH=linux_amd64` will work.

Reply:
done